### PR TITLE
hindent: Submission

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: textproc/hindent/Portfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                hindent
+version             6.2.0
+revision            0
+license             restrictive/distributable
+maintainers         nomaintainer
+platforms           darwin
```
Comment (reneeotten) [inline textproc/hindent/Portfile:10]: please remove, this is the default

File: textproc/hindent/Portfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                hindent
+version             6.2.0
+revision            0
+license             restrictive/distributable
```
Comment (reneeotten) [inline textproc/hindent/Portfile:8]: please capitilize as is done is done in (most) other ports

File: textproc/hindent/Portfile (hunk 1 above)
Comment (essandess) [reply textproc/hindent/Portfile:10]: Thanks done.

File: textproc/hindent/Portfile (hunk 2 above)
Comment (essandess) [reply textproc/hindent/Portfile:8]: Okay, but this is how it appears in the official [source](https://trac.macports.org/wiki/PortfileRecipes) [file](https://github.com/macports/macports-infrastructure/blob/master/jobs/distributable_lib.tcl#L71).

https://github.com/macports/macports-infrastructure/blob/300f8a312d3ffe7c161967afdfda053a38739927/jobs/distributable_lib.tcl#L71

**Pull request merged**